server/src/main/java/org/candlepin/controller/CandlepinPoolManager.java

```diff
@@ -1586,10 +1567,10 @@ else if (!poolQuantities.keySet().equals(entitlements.keySet())) {
         }
 
         handler.handlePostEntitlement(this, consumer, entitlements);
-        handler.handleSelfCertificates(consumer, poolQuantities, entitlements, generateUeberCert);
+        handler.handleSelfCertificates(consumer, poolQuantities, entitlements);
```

Comment:
They are too far out of band to care about.

Any old ueber certs should get cleaned up as part of this PR and any old certs should be ignored IF they happen to escape the cleanup included with this PR. Any new requests to get/generate uber certs will check the new table and generate in there.

Essentially, once this version of CP is deployed, ueber cert gen will start from scratch and be generated the new way. Today, any time satellite wants the ueber cert, it will attempt a GET and default to generate if it doesn't exist, so no worries from that side as well.